Sources/Fuzzilli/Core/JavaScriptEnvironment.swift

```diff
@@ -172,6 +173,8 @@ public class JavaScriptEnvironment: ComponentBase, Environment {
         }
         
         customPropertyNames = ["a", "b", "c", "d", "e"]
+        // This is needed in ProgramBuilder.generateVariable, such that method names don't collide with property names.
```

Comment:
I think we can just drop this comments, to me it seems intuitive that if we have `customPropertyNames`, then we should also have `customMethodNames`